pypi_formula_mappings.json

```
@@ -204,6 +204,9 @@
   "twarc": {
     "exclude_packages": ["six"]
   },
+  "wxpython": {
+    "exclude_packages": ["pillow", "six"]
```

```suggestion
    "exclude_packages": ["numpy", "pillow", "six"]
```
Currently:
```
➜ brew update-python-resources wxpython
==> Retrieving PyPI dependencies for "wxPython==4.1.1"...
==> Getting PyPI info for "numpy==1.21.0"
==> Excluding "pillow==8.2.0"
==> Excluding "six==1.16.0"
==> Excluding "wxpython==4.1.1"
==> Updating resource blocks
```
And a `numpy` resource block is added.

---

Done! Would be nice if we had a way to just automatically exclude things automatically when a formula has the right `depends_on`...